dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -33,4 +33,33 @@ def post_ui_tip_dismissed
 
     render status: 200, json: {set: property_name}
   end
+
+  # POST /api/v1/users/<user_id>/postpone_census_banner
+  def postpone_census_banner
+    today = Date.today
+    year = today.year
+    # Find the next scheduled date that is at least 2 weeks away
+    scheduled_display_dates = [1, 3, 5, 11].map {|month| Date.new(year, month, 15)}
+    scheduled_display_dates.push Date.new(year + 1, 1, 15)
+    next_date = scheduled_display_dates.select {|d| d > today && d > today + 2.weeks} [0]
+
+    @user.next_census_display = next_date
+    @user.save
+
+    render status: 200, json: {set: "next_census_display"}
```

Comment:
What is the point of this response body? Why is hardcoded `{set: "next_census_display"}` useful? Should we return the new census display time? Or no body `head 200` or `head :ok`

Reply:
Good point. I'm going to change it to return the new value.